python/cudf/cudf/_libxx/strings/char_types.pxd

```diff
@@ -8,14 +8,18 @@ cdef extern from "cudf/strings/char_types/char_types.hpp" \
         namespace "cudf::strings" nogil:
 
     ctypedef enum string_character_types:
-        DECIMAL = 1 << 0                              # binary 00000001
-        NUMERIC = 1 << 1                              # binary 00000010
-        DIGIT = 1 << 2                                # binary 00000100
-        ALPHA = 1 << 3                                # binary 00001000
-        SPACE = 1 << 4                                # binary 00010000
-        UPPER = 1 << 5                                # binary 00100000
-        LOWER = 1 << 6                                # binary 01000000
-        ALPHANUM = DECIMAL | NUMERIC | DIGIT | ALPHA  # binary 00001111
+        DECIMAL 'cudf::strings::string_character_types::DECIMAL' = 1 << 0
```

Comment:
We shouldn't define the `= 1 << 0` after this, just defining the C string literal should give us the value we want.